### PR TITLE
Component updates

### PR DESCRIPTION
**Pull request by nshahan**
Rollup of all internal changes since last release. No new components added.

Waiting for angular2 to release a revision before publishing these changes to pub. Requires analyzer 0.29.7 to build.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 name: angular2_components
-version: 0.3.1-alpha
+version: 0.4.0-alpha
```
Comment (kevmoo) [inline pubspec.yaml]: Because breaking changes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
+## 0.4.0-alpha
+
+This code is considered production quality, but depends on angular2: 3.0.0-alpha.
+The alpha tag represents the evolving nature of the AngularDart api, not
+code quality (3.0.0-alpha is used in production Google apps).
+
+Component update rollup.
```
Comment (kevmoo) [inline CHANGELOG.md]: This line...uh...how does it help?

File: pubspec.yaml (hunk 1 above)
Comment (nshahan) [inline pubspec.yaml]: Yes, just trying to highlight the fact that there are some breaking changes.

File: CHANGELOG.md (hunk 2 above)
Comment (nshahan) [inline CHANGELOG.md]: Not at all. It is just an artifact of how I was tracking changes. I will remove.

File: pubspec.yaml (hunk 1 above)
Comment (kevmoo) [inline pubspec.yaml]: Cool...

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+ * Focus: Fix AX_ARIA_08 a11y issue.
+ * Material Chips: Use :host to remove need for wrapper div.
+ * Material Expansionpanel: Fix panel overflow issues.
+ * Material Input:
+   * Add new number accessors and validators.
+   * Fix AX_TEXT_01 a11y issue.
+ * Material List: Block pointer events for disabled list items.
+ * Material Popup: Disable animation delay when there is nothing to animate.
+ * Material Radio: Fix styling issue, flex for IE11.
+ * Material Tab Panel: Fix issue that prevents displaying tabs on initialization.
```
Comment (alanrussian) [inline CHANGELOG.md]: Thank you for fixing this! I was running into this issue. Wish I saw this pending pull earlier. :-)

Comment (nshahan) [inline CHANGELOG.md]: In this case it was an easy fix so we went ahead and added it in. Thanks to @stephan-gruen for pointing out the fix.

**Comment by dejan-marlovic** [comment]
hi, can you please help me? I know that this is not the right place to ask for help about this but I could not find solution to this anywhere else 

 im trying to use:
<span tooltipTarget #ref="tooltipTarget">
  Explicitly declare a tooltip component<br/>
</span>

<material-tooltip-text [for]="ref">
    Allows for &nbsp; <strong>formatted</strong> &nbsp; <em>text</em>.
</material-tooltip-text>

and I get an error:
Transform TemplateCompiler on tooltip|lib/app_component.ng_meta.json threw error: Template parse errors:
line 4, column 21 of AppComponent: ParseErrorLevel.FATAL: There is no directive with "exportAs" set to "tooltipTarget"
#ref="tooltipTarget"

I can use other components fine

**Comment by nshahan** [comment]
I just opened an issue #59 to discuss.

**Pull request merged**